packages/typespec-autorest/test/openapi-output.test.ts

```diff
@@ -823,6 +823,161 @@ describe("typespec-autorest: extension decorator", () => {
   });
 });
 
+describe("typespec-azure: identifiers decorator", () => {
```

Comment:
Should add a test to verify that no extension is created when the object type contains a 'name' or 'id' property.